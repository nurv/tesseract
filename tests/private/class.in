class Example {
	int i = 0;
	int test(){
		i++;
		return i;
	}
}
Example j = new Example();
j.test()
j.test();
j.i
j.i = 1;
j.i++;
j.test();
int hello = 12;
class ExampleX extends ist.leic.pa.stack.Stack {
	int hello(){
		return hello--;
	}
}
ExampleX.getCurrentEvalLevel()
ExampleX ex = new ExampleX();
ex.hello()
ex.hello()
