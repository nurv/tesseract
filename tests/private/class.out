> > > 1
> > 2
> > > > > > "$Eval8"
> > 12
> 11
> 